packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.html

```diff
@@ -1,32 +1,43 @@
 <template>
-  <li class="slds-grid">
-    <div onclick={onSelect} class="facet__value-container facet__value-option slds-grid slds-grid_vertical-align-center slds-size_1-of-1 slds-p-around_x-small">
-      <template if:false={displayAsLink}>
-        <lightning-input 
-          type="checkbox"
-          class="facet__value-checkbox"
-          variant="label-hidden"
-          label="Facet value option"
-          name="facet value checkbox"
-          onchange={onSelect}
-          checked={isChecked}
-        ></lightning-input>
+  <div
+    role={divRole}
+    aria-checked={isChecked}
```

Comment:
for Facet checkbox, `aria-checked` is correct, but for Facet-display as link, I think `aria-pressed` makes more sense